dev/ComboBox/ComboBoxAttachedProperties.cpp

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <common.h>
+#include "ComboBoxAttachedProperties.h"
+#include "ComboBoxAttachedProperties.properties.h"
+#include "DispatcherHelper.h"
+#include "Converters.h"
+#include "HashMap.h"
+
+static constexpr auto c_popupName = L"Popup"sv;
+static constexpr auto c_popupBorderName = L"PopupBorder"sv;
+static constexpr auto c_editableTextName = L"EditableText"sv;
+
+using CornerRadiusFilterType = CornerRadiusFilterConverter::FilterType;
+
+// Normal ComboBox and editable ComboBox have different CornerRadius behaviors.
+// Xaml is not lifted yet when we implementing this feature so we don't have access to ComboBox code.
+// Creating this attached property to help us plug in some extra logic without touching the actual ComboBox code.
+void ComboBoxAttachedProperties::OnApplyDynamicCornerRadiusPropertyChanged(
+    const winrt::DependencyObject& sender,
+    const winrt::DependencyPropertyChangedEventArgs& args)
+{
+    static std::map<winrt::ComboBox*, std::vector<winrt::event_token>> comboBoxEventTokenMap;
```

Comment:
>comboBoxEventTokenMap [](start = 71, length = 21)

When you switch over to using weak references instead of raw pointers, consider having a 'cleanup' function that periodically goes through the list and removes any dead weakrefs. This will prevent the map from growing constantly.